TODO.md

```diff
@@ -0,0 +1,6 @@
+# TODO
+- [ ] Discuss splitting edge node provisioning, and edge application deployment into different labs
+    - EG; https://github.com/ahussey-redhat/device-edge-workshops focuses on the device deployment, and;
+    - https://github.com/redhat-na-ssa/rhte-edge-lab-sno focuses on application deployment
+    - This should reduce the overall time it takes to complete the full lab, as 90mins is not nearly enough time to; provision and configure edge clusters, and deploy applications
```

Comment:
A three hour engagement would be plentiful time 🙂 I understand RHTE was fairly constrained

That makes sense. I think it might be worth having a centralized `SNO` and/or `edge device` provisioning lab, which both this lab and the Device Edge workshop can reference